pkg/session_recording/recordings.jsx

```diff
@@ -238,17 +259,74 @@
                             navigateToItem={this.navigateToRecording.bind(this, r)}/>);
             }
             return (
-                <Listing.Listing title={_("Sessions")}
-                                 columnTitles={columnTitles}
-                                 emptyCaption={_("No recorded sessions")}
-                                 fullWidth={false}>
-                    {rows}
-                </Listing.Listing>
+                <div>
+                    <div className="content-header-extra">
+                        <table class="form-table-ct">
+                            <tr>
+                                <td className="top"><label className="control-label" for="dateSince">Date Since</label></td>
+                                <td>
+                                    <Datepicker date={dateSince} onDateChange={this.handleDateSinceChange} />
+                                </td>
+                                <td className="top"><label className="control-label" for="dateUntil">Date Until</label></td>
+                                <td>
+                                    <Datepicker date={dateUntil} onDateChange={this.handleDateUntilChange} />
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+                    <Listing.Listing title={_("Sessions")}
+                                     columnTitles={columnTitles}
+                                     emptyCaption={_("No recorded sessions")}
+                                     fullWidth={false}>
+                        {rows}
+                    </Listing.Listing>
+                </div>
             );
         }
     };
 
     /*
+     * A component representing a datepicker based on bootstrap-datepicker.
+     * Requires jQuery and bootstrap-datepicker.
+     * Properties:
+     * - onDateChange: function to call on date change event of datepicker.
+     */
+    let Datepicker = class extends React.Component {
```

Comment:
Even though evidently this works, it still makes me a bit nervous. Could you please move this class above the one that uses it, i.e. RecordingList? Thank you.